meson.build

```diff
@@ -440,6 +448,22 @@ executable('ratbagd',
 
 install_man('ratbagd/ratbagd.8')
 
+#### ratbagd_devel ####
+
+executable('ratbagd_devel',
+	   src_ratbagd,
+	   dependencies : deps_ratbagd,
+	   include_directories : include_directories('src'),
+	   install : false,
+	   c_args : '-DRATBAG_DBUS_INTF="ratbag_devel1_@0@"'.format(ratbagd_sha),
```

Comment:
I would like to buy a few letters so we can spell out INTERFACE. Storage is cheap these days, we can afford a few more characters that we never have to type anyway.